### PR TITLE
Implement Typescript

### PR DESCRIPTION
**Pull request by iObrovac**
@kiljee

### CODE REVIEW

**Comment by iObrovac** [comment]
@kiljee

**Comment by iObrovac** [comment]
@damjanbanjac

File: src/components/modals/NewAccModal.tsx [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
     dob: "",
     email: "",
     password: "",
   });
 
-  const { loggedIn, setLoggedIn, userData, setUserData } =
-    useContext(UserContext);
+  const { setLoggedIn, setUserData } = useContext<IContext>(UserContext);
 
   let navigate = useNavigate();
```
Comment (damjanbanjac) [inline src/components/modals/NewAccModal.tsx:20]: use const instead of let here.

File: src/pages/favorites/Favorites.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+const Favorites: React.FC = (): JSX.Element => {
+  const { loggedIn } = useContext<IContext>(UserContext);
+  const [flowers, setFlowers] = useState<ISingleFlower[]>();
+
+  useEffect(() => {
+    const getFavorites = async (): Promise<void> => {
+      try {
+        const response = await axios.get<IGetFavorites>(
+          `https://flowrspot-api.herokuapp.com/api/v1/flowers/favorites?page=1`,
```
Comment (damjanbanjac) [inline src/pages/favorites/Favorites.tsx:17]: Maybe to create some const folder, and remove there base url for backend api

File: src/pages/flowers/Flowers.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+const Flowers: React.FC = (): JSX.Element => {
+  const [flowerData, setFlowerData] = useState<IExampleFlower[]>();
+  const [favorites, setFavorites] = useState<ISingleFlower[]>();
+
+  const { loggedIn } = useContext<IContext>(UserContext);
+
+  const getFavorites = async (): Promise<void> => {
+    try {
+      const response2 = await axios.get<IGetFavorites>(
+        `https://flowrspot-api.herokuapp.com/api/v1/flowers/favorites?page=1`,
```
Comment (damjanbanjac) [inline src/pages/flowers/Flowers.tsx:20]: Use constant  for base api url

**Pull request merged**